tests/test_universe.py

```diff
@@ -72,7 +72,7 @@ def test_degree2(self):
         def _check(t):
             self.assert_allclose((1 - t) ** 2, f(t, 0, 2))
             self.assert_allclose(2 * (1 - t) * t, f(t, 1, 2))
-            self.assert_allclose(t ** 2, f(t, 2, 2))
+            self.assert_allclose(t**2, f(t, 2, 2))
```

Comment:
Do not make unnecessary changes.  Do not mix format change with anything else.  Please revert the format change.